systest/backup/advanced-scenarios/127-Namespace/docker-compose.yml

```diff
@@ -0,0 +1,86 @@
+# Auto-generated with: [./compose -a 3 -z 1 -r 1 -w --port_offset=0 --expose_ports=false --alpha_volume=./data/backups:/data/backups/ --zero_volume=./data/backups:/data/backups/ --mem= --names=false -O ../systest/backup/multi-tenancy/docker-compose.yml --acl]
+#
+version: "3.5"
+services:
+  alpha1:
+    image: dgraph/dgraph:local
+    working_dir: /data/alpha1
+    depends_on:
+      - zero1
+    labels:
+      cluster: test
+    ports:
+    - 8080
+    - 9080
+    volumes:
+    - type: bind
+      source: $GOPATH/bin
+      target: /gobin
+      read_only: true
+    - type: bind
+      source: ../../../acl/restore/acl-secret
+      target: /secret/hmac
+      read_only: true
+    - data-volume:/data/backups/
+
+    command: /gobin/dgraph  ${COVERAGE_OUTPUT} alpha --my=alpha1:7080 --zero=zero1:5080
+      --logtostderr -v=2 --raft "idx=1; group=1;" --security "whitelist=10.0.0.0/8,172.16.0.0/12,192.168.0.0/16;"
+      --acl "secret-file=/secret/hmac;"
+  
+  zero1:
+    image: dgraph/dgraph:local
+    working_dir: /data/zero1
+    labels:
+      cluster: test
+    ports:
+    - 5080
+    - 6080
+    volumes:
+    - type: bind
+      source: $GOPATH/bin
+      target: /gobin
+      read_only: true
+    - data-volume:/data/backups/
```

Comment:
can you leave a space after colon just like we do everywhere else?

Reply:
I reconfirmed online. Shared volumes are written like this.